README.md

```diff
@@ -292,14 +308,15 @@ Provide multiple paths/patterns:
 
 ```js
 {
-  title: 'Multiple Files',  
+  title: 'Multiple Files',
   files: [
     'my/project/styles/global/*',
     'my/project/styles/components/feedback.scss',
     '!my/project/styles/global/colors.scss'
   ]
 }
 ```
+*`!` negates the match*
```

Comment:
I think we need one or the other. I agree that linking to globby is substantial enough. Let's remove line 319.